zkvm/docs/zkvm-spec.md

```diff
@@ -998,9 +1005,8 @@ Code | Instruction                | Stack diagram                              |
 0x1d | [`nonce`](#nonce)          |    _pred blockid_ → _contract_             | Modifies [tx log](#transaction-log)
 0x1e | [`log`](#log)              |            _data_ → ø                      | Modifies [tx log](#transaction-log)
 0x1f | [`signtx`](#signtx)        |        _contract_ → _results..._           | Modifies [deferred verification keys](#transaction-signature)
-0x20 | [`call`](#call)            |_contract bf prog_ → _results..._           | [Defers point operations](#deferred-point-operations)
-0x21 | [`select:n:k`](#select)    | _contract x0...xn-1_ → _contract’_         | [Defers point operations](#deferred-point-operations)
-0x22 | [`delegate`](#delegate)    |_contract prog sig_ → _results..._          | [Defers point operations](#deferred-point-operations)
+0x20 | [`call`](#call)            |_contract(P) prf prog_ **call** → _results_ | [Defers point operations](#deferred-point-operations)
```

Comment:
```suggestion
0x20 | [`call`](#call)            |_contract(P) prf prog_ → _results_ | [Defers point operations](#deferred-point-operations)
```

we do not mention instruction name in these diagrams because it's already mentioned right there in the left column.

Reply:
`prf` usually stands for "pseudo-random function", so i'd use `proof` here.